### DISCUSSION TITLE
downgrade further

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#1239] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so like before im now on v125 thats the diffidence so can i downgrade further then v125 on kernel v4 on a dedde board

**Comment by Balistic123** [comment, marked as answer]
No you cant. v125 is the farthest you can go on kernver 4 unless you are unenrolled then you can use kvs to change your kernver

**Reply by buddyted** [reply]
So um about that I just managed to downgrade to v120 should this happen

**Reply by Balistic123** [reply]
No. That is interesting can you explain what you did?

**Reply by buddyted** [reply]
um the chromebook wont turn on now
